lib/modules/manager/index.ts

```diff
@@ -10,21 +12,24 @@ import type {
   Result,
 } from './types';
 export { hashMap } from './fingerprint.generated';
-const managerList = Array.from(managers.keys());
+const managerList = Array.from(managers.keys()); // does not include custom managers
+const customManagerList = getCustomManagerList();
+export const allManagersList = [...managerList, ...customManagerList];
 
 export function get<T extends keyof ManagerApi>(
   manager: string,
   name: T
 ): ManagerApi[T] | undefined {
-  return managers.get(manager)?.[name];
+  return managers.get(manager)?.[name] ?? customManagers.get(manager)?.[name];
 }
 export const getManagerList = (): string[] => managerList;
 export const getManagers = (): Map<string, ManagerApi> => managers;
 
 export async function detectAllGlobalConfig(): Promise<GlobalManagerConfig> {
   let config: GlobalManagerConfig = {};
-  for (const managerName of managerList) {
-    const manager = managers.get(managerName)!;
+  for (const managerName of [...managerList, ...customManagerList]) {
+    const manager =
+      managers.get(managerName)! ?? customManagers.get(managerName)!;
```

Comment:
maybe add a wrapper function for get manager which can be reused at multiple places?

Reply:
This is the only file where it is used like this, so adding it won't do much benefit.